nano/node/blockprocessor.cpp

```diff
@@ -67,7 +67,7 @@ void nano::block_processor::add (std::shared_ptr<nano::block> block_a, uint64_t
 	add (info);
 }
 
-void nano::block_processor::add (nano::unchecked_info const & info_a)
+void nano::block_processor::add (nano::unchecked_info const & info_a, bool push_front_preference)
```

Comment:
Minor but probably want `push_front_preference` to have a trailing `_a` to be consistent with the other parameter

Reply:
Makes sense, thanks!